Makefile

```diff
@@ -1,16 +1,11 @@
 SHELL=/bin/bash -o pipefail
-.PHONY: clean docker manual
+.PHONY: clean manual
 .DEFAULT_GOAL := manual
 
 clean:
 	rm -rf src/version.inc
 	rm -rf bin/
 
-docker:
```

Comment:
This is still useful for local development (it allows building and testing Wattsi without having a Free Pascal compiler). I'd like to keep it, and if possible, reuse it in the GitHub action too.

Reply:
Reusing will mean we can't use https://github.com/docker/build-push-action, but if this should stay that's OK, less stuff overall is better.

Reply:
I'd be OK using build-push-action but also keeping this for local development.

Reply:
I landed with using `docker run "$IMAGE_NAME" --version` in between the build and push, so that wouldn't be possible to fit into build-push-action.